File: javascript/1295-find-numbers-with-even-number-of-digits.js
/**
 * @param {number[]} nums
 * @return {number}
 */
 var findNumbers = function(nums) {
    let cnt = 0
    for (let i = 0; i < nums.length; i++) {
        if (`${nums[i]}`.length % 2 === 0) cnt++
    }
    return cnt
};

File: golang/1295-find-numbers-with-even-number-of-digits.go
package golang

func findNumbers(nums []int) int {
	count := 0
	for _, num := range nums {
		digits := 0
		for num > 0 {
			num = num / 10
			digits++
		}
		if digits%2 == 0 {
			count++
		}
	}
	return count
}
